docs/_docs/1_1_spark_quick_start_guide.md

```diff
@@ -300,6 +300,221 @@ spark.
   show(100, false)
 ```
 
+# Spark-Sql example
+## Setup
+Hudi support using spark sql to write and read data with the **HoodieSparkSessionExtension** sql extension.
+```shell
+# spark sql for spark 3
+spark-sql --packages org.apache.hudi:hudi-spark3-bundle_2.12:0.8.0,org.apache.spark:spark-avro_2.12:3.0.1 \
```

Comment:
0.8.0 does not support spark sql? move to 0.9.0-SNAPSHOT and would update the version when releasing.

Reply:
Yes, will fix this.